spec/features/scores/score_edit_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'As a registered user' do
+  before(:each) do
+    @user = create(:user)
+    allow_any_instance_of(ApplicationController).to receive(:current_user)
+                                                .and_return(@user)
+
+    json_user_resp = File.read('spec/fixtures/flat/user.json')
+    stub_request(:get, "https://api.flat.io/v2/me").to_return(status: 200, body: json_user_resp, headers: {})
+
+    json_score_resp = File.read('spec/fixtures/flat/create_score/users_scores_create_score.json')
+    stub_request(:get, "https://api.flat.io/v2/users/me/scores").to_return(status: 200, body: json_score_resp, headers: {})
+
+    visit users_dashboard_index_path
+  end
+
+  it 'I can edit scores' do
```

Comment:
this test is very bare bones, because most of the functionality is happening through javascript, so not sure how to test it